xla/stream_executor/cuda/BUILD

```diff
@@ -244,9 +248,18 @@ cc_library(
         "//xla/stream_executor:host_or_device_scalar",
         "//xla/stream_executor:stream_executor_headers",
         "//xla/stream_executor/platform",
+    ]) + if_rocm_is_configured([
+        "@local_config_rocm//rocm:rocm_headers",
     ]) + ["@tsl//tsl/platform:errors"],
 )
 
+cc_library(
+    name = "cublas_lt_header_for_rocm",
+    hdrs = ["cuda_blas_lt.cc", 
```

Comment:
Please do not put .cc files as headers. If it is a header, it should be a .h file.